backend/src/main/resources/application.yaml

```diff
@@ -9,6 +9,7 @@ user-management-client:
     manage-users: manage-users
 config:
   vanity-hostname: localhost
+  allowed-origins: "http://local.umc.com:8080"
```

Comment:
These settings need to be modified locally to make things work. I was using `/etc/hosts`/ but I typical set-up would just be `*`.